### DISCUSSION TITLE
Is there any way to reuse a page template across tenants?

### DISCUSSION THREAD

**Discussion by ravimpatel** [OrchardCMS/OrchardCore#9999] [Q&A]
Page templates can be used to create pages within a tenant. However, is it possible to define a global page template which can reused across tenants? If so, how? Also, is it possible to modify the said "global page template" so that it reflects changes across all tenants?

**Comment by ns8482e** [comment]
Define your views in application

**Reply by ravimpatel** [reply]
I am talking about the ability to edit page template online (without re-deployment). Can the approach you described achieve that?

**Reply by ns8482e** [reply]
No, Template feature is tenant specific and it only applies to the tenant where it's defined.
